e3db/types/search_range.py

```diff
@@ -1,61 +1,94 @@
+from datetime import timezone, timedelta, datetime
+import re
 
 class Range():
-    def __init__(self, key="CREATED", format="Unix", zone="UTC", zone_offset=None, start=None, end=None):
+    def __init__(self, key="CREATED", start=None, end=None, zone_offset=None):
         """
         Initialize the Range class for use in Search. This class can be manually created if needed, but 
         the Search.range() method handles the Search workflow.
 
+        Once created, modifying zone_offset will not propogate the timezone changes over to start and end.
+        If you wish to change the zone for a particular time within the same Range object, provide a datetime object
+        with zone information (tzinfo).
+
         Parameters
         ----------
         key : str, optional
             "CREATED|MODIFIED" (the default is "CREATED")
 
-        format : str, optional
-            Currently is not a supported parameter (the default is "Unix")
-
-        zone : str, optional
-            Since python time objects are naive or zone-agnostic, this will attempt to append
-            the proper timezone information to the time object for the query.
-            Currently supported are:
-                "PST":"-08:00", "MST":"-07:00", "CST":"-06:00", "EST":"-05:00", "UTC":"+00:00"
-                (the default is "UTC" if the proper timezone cannot be found, which represents +00:00)
-
-        zone_offset : str, optional
-            If provided this offset will be used over zone.
-            Accepts the format "[+|-]dd:dd"
-            (the default is None(UTC), which will attempt to use zone if provided)
+        start: datetime, int, optional
+            Search only for record that come after this time 
+            Accepts datetime object with and without timezone information, see zone_offset for more details.
+            OR
+            Accepts int denoting unix epoch time and this will always be in UTC timezone.
```

Comment:
Specify epoch seconds because results will look weird if they start providing millis 😝 . Technically it is not in a timezone, might drop that phrasing.